src/services/notifications.rs

```diff
@@ -0,0 +1,60 @@
+use crate::models::backend::notifications::NotificationRegistrationRequest as BackendRegistrationRequest;
+use crate::models::service::notifications::{
+    DeviceData, NotificationRegistrationRequest, SafeRegistration,
+};
+use crate::providers::info::{DefaultInfoProvider, InfoProvider};
+use crate::utils::context::Context;
+use crate::utils::errors::ApiResult;
+
+pub async fn delete_registration(
+    context: Context<'_>,
+    chain_id: String,
+    uuid: String,
+    safe_address: String,
+) -> ApiResult<()> {
+    let client = context.client();
+
+    let info_provider = DefaultInfoProvider::new(&chain_id, &context);
+    let url = core_uri!(
+        info_provider,
+        "/v1/notifications/devices/{}/safes/{}/",
+        uuid,
+        safe_address
+    )?;
+
+    client.delete(url).send().await?;
```

Comment:
Should we check the status here, to indicate if an unsubscribe was successful, or do the clients ignore the status code of this request? (e.g. would the client block the removal of a Safe if it cannot unsubscribe)

Reply:
I would merge the issue and consult with the clients if the errors are meaningful to them.